config/deploy/stage.rb

```diff
@@ -0,0 +1,6 @@
+ENV['APP_ENV']  ||= 'staging'
+require_relative 'server_settings'
+
+server 'sdr-services-test.stanford.edu', user: 'sdr2service', roles: %w{app}
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
So, one way to simplify all that is for the sdr-services-dev box to use MySQL, but that requires a lot more coordination with dev-ops and sys-ops to make that change, AFAICT.  If that were done, then the `local` and the `integration` groups could be folded into one `development` group (hope that makes sense).